notarysql/postgresql-initdb.d/tls-setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Setup the server so it knows where to find certs so that server can be
+# started with TLS enabled.
+set -e
+
+sed -i "s/#ssl = off/ssl = on/" "$PGDATA"/postgresql.conf
```

Comment:
It is set in the Dockerfile for the image: https://github.com/docker-library/postgres/blob/master/9.5/Dockerfile#L60